packages/client/src/components/Header/Header.tsx

```diff
@@ -369,6 +370,12 @@ class HeaderComp extends React.Component<IFullProps, IState> {
         placeHolderText: intl.formatMessage(messages.placeHolderNationalId)
       })
     }
+    searchTypeList.push({
```

Comment:
We just need to add a conditional here checking whether the "registrationEmail" field is actually included in the form or not